CONTRIBUTING.md

```diff
@@ -344,7 +320,7 @@ To publish to PyPI:
 3. Run
 
 ```bash
-poetry publish --build
+poetry publish
```

Comment:
Still running `poetry` for publishing?

I've been used to running `twine upload dist/*` as a standalone way. Perhaps `uv` has some option for this?

Reply:
Ah this is outdated, I switched to `uv publish` but haven't had the chance to test this yet

Reply:
Can we test this without directly creating a new release 😅